inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -728,6 +728,9 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         LayerParams attrs = {node->get_friendly_name(), "DetectionOutput",
                             details::convertPrecision(node->get_output_element_type(0))};
         auto res = std::make_shared<InferenceEngine::CNNLayer>(attrs);
+        if (res == nullptr) {
```

Comment:
make_shared could not return nullptr. It throws exception instead.

Reply:
The same below.

Reply:
thanks, fixed